### ISSUE TITLE
Generic blacklist g

### ISSUE THREAD

**Issue by r-darwish** [r-darwish/topgrade#105]

**Comment by nCrazed** [comment]
Is this intended to be a replacement for the specific `--no-system`, `--not-git-repos`, `--no-vim`, `--no-emacs` flags, or something else entirely?

**Comment by r-darwish** [comment]
Yeah, I think it makes more sense that we'll have a `--disable emacs,vim` instead of `--no-vim --no-emacs`, etc...
